src/io/imageutils-lodepng.cc

```diff
@@ -1,4 +1,4 @@
-#include "imageutils.h"
+#include "io/imageutils.h"
 #include "ext/lodepng/lodepng.h"
```

Comment:
Some places, "ext/" is included, other places not. Would be good to choose one style.
..plus the <> vs. "" discussion.

Reply:
Oh, I had only modified single `"foo.h"` type headers, haven't seen that there are so many `"somtimespartial-or-even-excessive-prefix/foo.h"` headers. Fixed now.

Same here: as long as we have it in src/ext and should be included from there always, they should be quote. To get them angle-bracketified, we need to make a choice in cmake and either pull the system header or from submodules/.

Reply:
ok, let's follow your suggestion,